packit_service/service/api/runs.py

```diff
@@ -0,0 +1,126 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from http import HTTPStatus
+from logging import getLogger
+
+from flask_restx import Namespace, Resource
+
+from packit_service.models import (
+    CoprBuildModel,
+    KojiBuildModel,
+    RunModel,
+    SRPMBuildModel,
+    TFTTestRunModel,
+    optional_time,
+)
+from packit_service.service.api.parsers import indices, pagination_arguments
+from packit_service.service.api.utils import get_project_info_from_build, response_maker
+
+logger = getLogger("packit_service")
+
+ns = Namespace("runs", description="Pipelines")
```

Comment:
Is this some convention that we name it `ns` each time and need to rename it when importing?..;)

Reply:
Probably because it's shorter in decorators afterwards? Otherwise have no clue